### PR TITLE
update efel package

### PR DESCRIPTION
**Pull request by genric**
@wizmer pls. FYI @wvangeit

### CODE REVIEW

File: patches/additionalPythonPackages/default.nix [hunk 1]
```diff
@@ -460,13 +461,13 @@ in
 
     };
 
-    efel = pythonPackages.buildPythonPackage rec {
+    efel = python27Packages.buildPythonPackage rec {
```
Comment (wizmer) [inline patches/additionalPythonPackages/default.nix]: What's the reason for this change ?

Comment (genric) [reply patches/additionalPythonPackages/default.nix]: efel is python 2(https://pypi.org/project/efel/) So I want to be explicit about that. Now we have efel python 3 nix package which doesn't work but it is still there.

Comment (wizmer) [reply patches/additionalPythonPackages/default.nix]: I am not even sure this works. Remove the change on this line and I'll merge

Comment (genric) [reply patches/additionalPythonPackages/default.nix]: done
